mediaflow_proxy/extractors/base.py

```
@@ -2,48 +2,79 @@
+        # - Redirect automatici attivi per seguire 301/302 (es. daddylive.sx -> thedaddy.top)
+        # - Timeout ragionevole; aggiungere altri parametri se servono (http2, proxies, ecc.)
+        self._default_client = httpx.AsyncClient(
+            timeout=httpx.Timeout(20.0),
+            follow_redirects=True,  # segue 3xx automaticamente [5][6]
+        )  # verify=True di default [3][2]
+
+        # Client "insecure" per domini con certificati problematici (es. newkso.ru)
+        # Valuta di sostituire verify=False con un SSLContext dedicato per maggiore sicurezza.
+        self._insecure_client = httpx.AsyncClient(
+            timeout=httpx.Timeout(20.0),
+            verify=False,
```

Ruff (0.12.2)

Probable use of `httpx` call with `verify=False` disabling SSL certificate checks

(S501)